web/src/bootstrap_typeahead.ts

```diff
@@ -794,6 +798,7 @@ export class Typeahead<ItemType extends string | object> {
                     // the search bar).
                     this.openInputFieldOnKeyUp();
                 }
+                this.clear_typeahead_tooltip?.();
```

Comment:
Why is this the right place for a second call? Why is the `default` block here something that we should expect to hide a typeahead tooltip?

It needs a comment.